packages/ui/src/components/HeaderClient/HeaderClient.tsx

```diff
@@ -0,0 +1,31 @@
+'use client'
+import { usePathname } from 'next/navigation'
+import Link from 'next/link'
+import { Icon } from '../Icon'
+
+export interface HeaderProps {
+  textStyle?: string
+  iconStyle?: string
+}
+export const HeaderClient = ({ iconStyle }: HeaderProps) => {
```

Comment:
There is no need of textStyle in the HeaderProps if its not sent to the component.